assets/src/scss/pages/_listing-page.scss

```diff
@@ -364,3 +364,15 @@
     }
   }
 }
+
+.listing-page-filters {
+  width: 100%;
+
+  .listing-page-select {
+    width: 220px;
```

Comment:
If the selected option value is longer than 220px, the arrow icon is covered:

![News-Stories-Greenpeace](https://github.com/user-attachments/assets/50e92592-2860-453b-bcff-1404d0d1c5bd)

Reply:
Good catch, fixed!